docs/getting-started/zowe_v2_faq.md

```diff
@@ -6,4 +6,30 @@ The official date is TBD, the target is April 25, 2022; look for the official an
 
-The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
\ No newline at end of file
+The Zowe Squads have prepared XLS spreadsheets with conformance criteria for all Zowe extensions including: CLI, APIs, App Framework, and Explorerfor VS Code. The spreadsheets clearly show the prior / V1 criteria alongside the new / V2 criteria. Please be aware, there are additions, deletions, and CHANGES to the criteria. In some cases the change is simply that a BEST PRACTICE has been deemed REQUIRED. Use the light-GREEN highlights to easily identify the changes. See the Changes to the [Conformance Criteria](https://www.zowe.org/vnext#conformance-changes) section at Zowe.org/vNext.
+
+## Whats the difference between "server.json" and "example-zowe.yaml"?
+
+The previous Zowe V1.x config, "server.json", has been removed from V2 and has been replaced with a new yaml configuration file. The app server will no longer support instances/workspaces which only contain a "server.json" config file and will fallback to a default configuration. In addition to the app server, ZSS will no longer support "server.json".
+
+The yaml Zowe configuration file contains configurations for the setup, install, and initialization of Zowe as well as for individual components. This file allows users to customize dataset names, security related configs, certificate setup/config, job name & job prefix, various runtime configs, high availability config, as well as individual component configurations.
+
+For more information on Zowe setup and the yaml configuration, run the following command in the command line:
+
+  zwe init --help
+
+## What are the new default ports?
+
+Two of the default Zowe ports have changed: the app server and zss. The new default app server port is 7556 (previously 8544) and the new zss port is 7557 (previously 8542).
+
+## How do I access Zowe through the API Mediation Layer in V2?
+
+In pervious V1.X versions of Zowe, the desktop could be acessed via the API Medation Layer by navigating to https:/${gateway_host}:{gateway_port}/ui/v1/zlux. In Zowe V2, the route to access the desktop has changed to https:/${gateway_host}:{gateway_port}/zlux/ui/v1. Such routing structure is applicable to other clients connected to the API Gateway. For example, the API Catalog may be accessed via https:/${gateway_host}:{gateway_port}/apicatalog/ui/v1.
+
+## What frameworks are supported in V2?
```

Comment:
Should be reworded to say "What new frameworks are supported in the App Framework in V2?"